code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -565,18 +568,23 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			fnft_collection_id: T::AssetId,
 			fnft_instance_id: T::FinancialNftInstanceId,
+			duration: DurationSeconds,
```

Comment:
We are not going to allow setting a duration, just that original duration gets applied again.